.github/ workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: Java CI
+
+on:
+  push:
+    branches:
+      - ci-setup
+  pull_request:
+    branches:
+      - ci-setup
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

Comment:
bump this version to  **actions/checkout@v4**  https://github.com/actions/checkout